account-kit/smart-contracts/src/ma-v2/actions/install-validation/installValidation.ts

```diff
@@ -149,9 +149,7 @@ export const installValidationActions: <
       throw new EntityIdOverrideError();
     }
 
-    const { encodeCallData } = account;
-
-    const callData = await encodeCallData(
+    return await account.encodeCallData(
```

Comment:
🌈 nit: no need to use `await` when returning a promise from an async function (same thing below)
```suggestion
    return account.encodeCallData(
```